CMakeLists.txt

```diff
@@ -210,13 +210,13 @@ ENDIF()
 #
 MESSAGE(STATUS "")
 MESSAGE(STATUS "Setting up muParser")
-IF(${IBAMR_FORCE_BUNDLED_MUPARSER})
-  SET(IBAMR_USE_BUNDLED_MUPARSER TRUE)
+IF(${IBAMR_FORCE_BUNDLED_muParser})
+  SET(IBAMR_USE_BUNDLED_muParser TRUE)
 ELSE()
-  FIND_PATH(MUPARSER_INCLUDE_DIRS NAMES muParser.h HINTS ${MUPARSER_ROOT}/include /usr/include/)
-  FIND_LIBRARY(MUPARSER_LIBRARIES NAMES muparser HINTS ${MUPARSER_ROOT}/lib /usr/lib)
-  IF("${MUPARSER_INCLUDE_DIRS}" STREQUAL "MUPARSER_INCLUDE_DIRS-NOTFOUND" OR
-      "${MUPARSER_LIBRARIES}" STREQUAL "MUPARSER_LIBRARIES-NOTFOUND")
+  FIND_PATH(muParser_INCLUDE_DIRS NAMES muParser.h HINTS ${muParser_ROOT}/include /usr/include/)
+  FIND_LIBRARY(muParser_LIBRARIES NAMES muparser HINTS ${muParser_ROOT}/lib /usr/lib)
+  IF("${muParser_INCLUDE_DIRS}" STREQUAL "muParser_INCLUDE_DIRS-NOTFOUND" OR
+      "${muParser_LIBRARIES}" STREQUAL "muParser_LIBRARIES-NOTFOUND")
     SET(IBAMR_USE_BUNDLED_muParser TRUE)
   ELSE()
     SET(IBAMR_USE_BUNDLED_muParser FALSE)
```

Comment:
The bug is on line 225: we should be using `muParser` here.

It is somewhat arbitrary, but muParser should have a CMake package so I am using its proper name instead of all caps (like I do for libmesh).

Reply:
What is the rationale for the difference between "proper name" and "always all caps"?

Reply:
Good question. I should make this consistent. I prefer all caps but I went with `Boost` and `Eigen3` since CMake uses those names in its own code - the result only makes sense if you know which modules CMake provides.

I'll write a followup commit that changes the defaults to be all-caps but also accepts common capitalizations (`libMesh`, `PETSc`, `Eigen`, etc) as alternatives.

Reply:
Fixed!